internal/image/ova.go

```diff
@@ -47,6 +47,31 @@ func (o *Ova) Validate() error {
 	return nil
 }
 
+func calculateTarSize(contentSize int) int {
+	const blockSize = 512
+
+	// Size of the tar header block
+	size := blockSize
+
+	// Size of the file content, rounded up to nearest 512 bytes
+	size += (contentSize + blockSize - 1) / blockSize * blockSize
```

Comment:
Isn't ` / blockSize * blockSize = 1`

Reply:
Not sure I understand. It could be `=1`. But only in case `contentSize + 2*blockSize - 1` = 262144.

Reply:
Oh, I know hat you mean. I should do:
```((contentSize + blockSize - 1) / blockSize) * blockSize```
to make it clear